c++/src/capnp/rpc.h

```diff
@@ -359,13 +359,13 @@ class RpcFlowController {
   // sends will propagate the exception.
   //
   // Note that messages sent with this method must still be delivered in the same order as if they
-  // had been sent with `message->send()`; they cannot be deloyed until later. This is important
+  // had been sent with `message->send()`; they cannot be deployed until later. This is important
```

Comment:
This really isn't supposed to be "delayed"?

Reply:
Oh geez, my internal spell correct is no better than your average computer apparently.